include/picongpu/fields/incidentField/profiles/DispersivePulse.hpp

```diff
@@ -299,15 +299,39 @@ namespace picongpu
                             float_X const dt = static_cast<float_X>(picongpu::SI::DELTA_T_SI / UNIT_TIME);
                             // interpolation order
                             float_X N_raw = Unitless::INIT_TIME / dt;
-                            int n = static_cast<int>(N_raw * 0.5_X); // -0 instead of -1 for rounding up N_raw
+                            int const n = static_cast<int>(N_raw * 0.5_X); // -0 instead of -1 for rounding up N_raw
+
+                            // frequency step for DFT
+                            float_X const dOmk = pmacc::math::Pi<float_X>::doubleValue / Unitless::INIT_TIME;
+
+                            // Since the (Gaussian) spectrum has only significant values near the central frequency,
+                            // the summation over all frequencies is reduced to frequencies within an interval
+                            // around the central frequency.
+
+                            // standard deviation of the Gaussian distributed spectrum
+                            float_X const sigma_Om = 1._X / (pmacc::math::sqrt(2._X) * Unitless::PULSE_DURATION);
+
+                            // index of the mean frequency of the Gaussian distributed spectrum
+                            // unit: [dOmk]
+                            int const center_k
+                                = static_cast<int>(SPEED_OF_LIGHT * Unitless::INIT_TIME / Unitless::Base::WAVE_LENGTH);
+
+                            // index of the lowest frequency in the Gaussian distributed spectrum which is used in the
+                            // DFT 4*sigma_Om distance from central frequency
+                            int const minOm_k = center_k - static_cast<int>(4._X * sigma_Om / dOmk);
```

Comment:
Do we want to hard-code the `4._X` here or should it not rather be a parameter in the `incidentField.param`?

Reply:
it should be part of the dispersive pulse

Reply:
Yes but it could be part of the `DispersivePulseParam` in case users want to play around with this value. Since it might confuse people, it could still have a default value of 4.